store/types/proof.go

```diff
@@ -88,68 +88,30 @@ func (op CommitmentOp) GetKey() []byte {
 // If length 0 args is passed in, then CommitmentOp will attempt to prove the absence of the key
 // in the CommitmentOp and return the CommitmentRoot of the proof
 func (op CommitmentOp) Run(args [][]byte) ([][]byte, error) {
+	// calculate root from proof
+	root, err := op.Proof.Calculate()
+	if err != nil {
+		return nil, sdkerrors.Wrapf(ErrInvalidProof, "could not calculate root for proof: %v", err)
```

Comment:
```suggestion
		return nil, sdkerrors.Wrapf(ErrInvalidProof, "could not calculate root for proof: %w", err)
```
why not wrap the error as well?

Reply:
this doesn't work because `sdkerrors.Wrapf` uses `fmt.Sprintf` and not `fmt.Errorf`